tasks/5.js

```diff
@@ -0,0 +1,10 @@
+function wordSearch(w) {
+  var string = "Hello World".toLowerCase();
+  w = w.toLowerCase();
+
+  return string.includes(w);
+}
```

Comment:
Consider that the task requires a full word search. `wordSearch("hell')` should return false